Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN INSTALL_PKGS=" \
     chmod og+w /tmp/_working_dir
 COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/_output/bin/cluster-logging-operator /usr/bin/
 COPY scripts/* /usr/bin/scripts/
-COPY files/ /usr/bin/files/
+COPY files/ /etc/clo/
```

Comment:
what does this buy us? we will no longer have `files` under our `workdir`

Reply:
I think this change is based on this review comment:
https://github.com/openshift/cluster-logging-operator/pull/112
Here's the new path:
fluentdAlertsFile = "/etc/clo/fluentd_prometheus_alerts.yaml"